tests/rados/pool_tests.py

```diff
@@ -514,6 +515,11 @@ def run(ceph_cluster, **kw):
         log.debug("Verifying pg_num_min on pools")
         target_configs = config["verify_pg_num_min"]["configurations"]
         try:
+            log.debug(
+                "Setting the default autoscaler mode to on, making sure that the Pool created will have "
+                "PG Autoscaler enabled on it."
+            )
+            rados_obj.configure_pg_autoscaler(default_mode="on")
```

Comment:
Why is this something that we need to do? The default value is anyway ON, right?

Reply:
Yes, the default is on. This is to avoid false errors/ clean-up misses from previous tests. 

No harm or foul done, as it's the default behaviour anyway.